### DISCUSSION TITLE
Executing scripts in @script does not work (x-on:click)

### DISCUSSION THREAD

**Discussion by rstanislav** [livewire/livewire#7789] [7. Bugs (archived)]
### Livewire version

v3.4.0

### Laravel version

v10.42.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

Looks like related issue: https://github.com/livewire/livewire/discussions/7348
x-on:click is not working in latest version is it related or new issue ?
example: https://wirebox.app/b/47r1n
if javascript wrapped in Alpine.data('fName', () => {
            return {
            log() {
        alert('it works');
    }
}))
then it works, it also works without @script @endscript tags
Dont know how to write failing test :(( so only photo: 

![IMG_3140](https://github.com/livewire/livewire/assets/1781042/6d753335-7518-46cb-9159-4dadc7aa9883)

### Repository or Wirebox URL

https://wirebox.app/b/47r1n

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ilyam2014** [comment]
aslo an empty script tag gives the following error
![image](https://github.com/livewire/livewire/assets/89526742/f1e9a1ad-79df-4dfb-acba-33f9afba0a27)
![image](https://github.com/livewire/livewire/assets/89526742/82c5a96a-515d-4a45-a5b6-9ec053881ece)

**Reply by calebporzio** [reply]
Ooof this looks like a bug. Can you submit a failing test PR for this case?

**Reply by rstanislav** [reply]
I have noticed it too, after v3.4.0 update, but this error (with missing "}" symbol) appears in random places across all projects(also without @script @endscript tags)

**Comment by sweebee** [comment]
Same issue here with 3.4:

```
Alpine Expression Error: Unexpected token 'var'

Expression: "<!--
    var options = {"chart":{"type":"line","height":500,"width":"100%","toolbar":{"show":true,"export":{"csv":{"headerCategory":"Dag"}}},"stacked":false,"zoom":{"enabled":true},"fontFamily":"ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji","foreColor":"#373d3f","defaultLocale":"nl","locales":[{"name":"nl","options":{"shortMonths":["jan","feb","maa","apr","mei","jun","jul","aug","sep","okt","nov","dec"]}}]},"plotOptions":{"bar":{"horizontal":false}},"colors":["#008FFB","#00E396","#feb019","#ff455f","#775dd0","#80effe","#0077B5","#ff6384","#c9cbcf","#0057ff","#00a9f4","#2ccdc9","#5e72e4"],"series":[{"name":"Vlag","type":"line","data":[],"options":[],"undefinedColor":"#22292F"}],"dataLabels":{"enabled":false},"labels":["2024-01-01 GMT","2024-01-02 GMT","2024-01-03 GMT","2024-01-04 GMT","2024-01-05 GMT","2024-01-06 GMT","2024-01-07 GMT","2024-01-08 GMT","2024-01-09 GMT","2024-01-10 GMT","2024-01-11 GMT","2024-01-12 GMT","2024-01-13 GMT","2024-01-14 GMT","2024-01-15 GMT","2024-01-16 GMT","2024-01-17 GMT","2024-01-18 GMT","2024-01-19 GMT","2024-01-20 GMT","2024-01-21 GMT","2024-01-22 GMT","2024-01-23 GMT","2024-01-24 GMT","2024-01-25 GMT","2024-01-26 GMT","2024-01-27 GMT","2024-01-28 GMT","2024-01-29 GMT","2024-01-30 GMT","2024-01-31 GMT"],"title":{"text":[]},"subtitle":{"text":"","align":"left"},"xaxis":{"labels":{"datetimeUTC":false},"categories":[],"type":"datetime"},"grid":{"show":true},"markers":{"size":4,"colors":["#008FFB","#00E396","#feb019","#ff455f","#775dd0","#80effe","#0077B5","#ff6384","#c9cbcf","#0057ff","#00a9f4","#2ccdc9","#5e72e4"],"strokeColors":"#fff","strokeWidth":2,"hover":{"size":7}},"stroke":{"show":true,"width":4,"colors":["#008FFB","#00E396","#feb019","#ff455f","#775dd0","#80effe","#0077B5","#ff6384","#c9cbcf","#0057ff","#00a9f4","#2ccdc9","#5e72e4"],"curve":"straight"},"tooltip":{"x":{"format":"d MMM yyyy"}}};

    var chart_mivuchnrljosktxbypzegaqwf = new ApexCharts(document.querySelector("#mivuchnrljosktxbypzegaqwf"), options);

    chart_mivuchnrljosktxbypzegaqwf.render();
//-->"
```

**Reply by joshhanley** [reply]
@sweebee any chance yuo could submit a failing test PR that replicates the issue? Thanks!
